README.md

```diff
@@ -1,11 +1,16 @@
 # The SpatialOS Games Development Kit for Unreal (alpha)
->This is an [alpha (SpatialOS website)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
+The SpatialOS Games Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
+
+The GDK offers:
+* **Multi-server support**: leveraging our cloud platform SpatialOS (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
+* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
```

Comment:
Technically there is no "character movement controller"

We can just say "character movement". Also can add stuff like ability system maybe, although it doesn't work across boundaries yet.